akka-docs/rst/java/http/routing-dsl/directives/alphabetically.rst

```diff
@@ -119,7 +119,7 @@ Directive                                   Description
                                             ``immutable.Seq[HttpHeader] ⇒ immutable.Seq[HttpHeader]`` function
 :ref:`-mapRouteResult-java-`                     Transforms the ``RouteResult`` with a ``RouteResult ⇒ RouteResult``
                                             function
-:ref:`-mapRouteResultFuture-java-`               Transforms the ``RouteResult`` future with a
+:ref:`-mapRouteResultStage-java-`               Transforms the ``RouteResult`` future with a
```

Comment:
The tricky bit is that we accept CompletionStage everywhere, but people actually will pass-in CompletableFuture most of the time. So putting full name is tricky... Maybe let's keep it Future as java people will realise it's that one?